source/screens/caseScreens/CaseSummary.tsx

```diff
@@ -42,6 +42,7 @@ const {
   ACTIVE_SIGNATURE_PENDING,
   ACTIVE_COMPLETION_REQUIRED_VIVA,
   ACTIVE_RANDOM_CHECK_REQUIRED_VIVA,
+  ACTIVE_COMPLETION_SUBMITTED,
```

Comment:
We need to add `ACTIVE_RANDOM_CHECK_SUBMITTED` as well. This is the good old 'Stickprovskontroll'